Rename library when built with Qt5 for co-installability.  Discussed with
upstream, but not reviewed as they aren't particularly interested in it.
Index: qscintilla2-2.8.4+dfsg/Qt4Qt5/qscintilla.pro
===================================================================
--- qscintilla2-2.8.4+dfsg.orig/Qt4Qt5/qscintilla.pro	2014-09-21 22:25:27.974348084 -0400
+++ qscintilla2-2.8.4+dfsg/Qt4Qt5/qscintilla.pro	2014-09-21 22:25:27.958348085 -0400
@@ -41,6 +41,7 @@
 
 greaterThan(QT_MAJOR_VERSION, 4) {
 	QT += widgets printsupport
+	TARGET = qt5scintilla2
 
     greaterThan(QT_MINOR_VERSION, 1) {
 	    macx:QT += macextras
Index: qscintilla2-2.8.4+dfsg/Qt4Qt5/features/qscintilla2.prf
===================================================================
--- qscintilla2-2.8.4+dfsg.orig/Qt4Qt5/features/qscintilla2.prf	2014-09-21 22:25:27.974348084 -0400
+++ qscintilla2-2.8.4+dfsg/Qt4Qt5/features/qscintilla2.prf	2014-09-21 22:25:27.966348085 -0400
@@ -17,9 +17,18 @@
         win32: {
             LIBS += -lqscintilla2d
         } else {
-            LIBS += -lqscintilla2
+            greaterThan(QT_MAJOR_VERSION, 4) {
+                LIBS += -lqt5scintilla2
+            } else {
+                LIBS += -lqscintilla2
+            }
         }
     }
 } else {
+    greaterThan(QT_MAJOR_VERSION, 4) {
+        LIBS += -lqt5scintilla2
+        }
+        else {
     LIBS += -lqscintilla2
+        }
 }
Index: qscintilla2-2.8.4+dfsg/designer-Qt4Qt5/designer.pro
===================================================================
--- qscintilla2-2.8.4+dfsg.orig/designer-Qt4Qt5/designer.pro	2014-09-21 22:25:27.974348084 -0400
+++ qscintilla2-2.8.4+dfsg/designer-Qt4Qt5/designer.pro	2014-09-21 22:25:27.966348085 -0400
@@ -11,8 +11,11 @@
 
     # Work around QTBUG-39300.
     CONFIG -= android_install
+    TARGET = qt5scintillaplugin
+    LIBS += -lqt5scintilla2
 } else {
     CONFIG += designer
+    LIBS += -lqscintilla2
 }
 
 macx {
